src/include/detail/graph/dfs.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file   dfs.h
```

Comment:
(Minor) Let's not check this in if it's just an empty file.

Reply:
@lums658 it looks like this was not deleted, though this comment was marked resolved. Could we delete? 

Also, in the future, would you mind leaving all my comments unresolved? I will go through them myself and mark as resolved. This helps avoid situations like this where things are incorrectly marked as resolved.

Reply:
Roger.